optimum/intel/__init__.py

```diff
@@ -191,6 +191,7 @@
             "OVModelOpenCLIPVisual",
             "OVModelOpenCLIPText",
             "OVModelOpenCLIPForZeroShotImageClassification",
+            "OVModelForZeroShotImageClassification",
```

Comment:
why do we need `OVModelOpenCLIPForZeroShotImageClassification` then?

Reply:
for open_clip library support and its models, new one added for transformers. open clip has different API (splitting models on 2 parts, different configuration approach and pre- and post- processing steps

Reply:
well, I can't find `AutoModelOpenCLIPForZeroShotImageClassification`. How it is loaded? I thought the concept of moving to OpenVINO backend is always replacing `AutoModel` to `OvModel`.